test/generate-json.py

```diff
@@ -15,7 +15,7 @@
 # PingMessage is generated by this project
 import sys
 sys.path.append("../src/protocol/python")
-import PingMessage as Message
```

Comment:
thank goodness, I was telling myself this couldn't be right, so annoying to prepend everythign